antiplag/serializers.py

```diff
@@ -64,4 +64,4 @@ class DocumentResultSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Document
-        fields = ("name", "result")
+        fields = ("name", "result", "text")
```

Comment:
I saw you made changes on another branch. Can you please remove it from this pull request?